src/js/components/aboutTheData/AgencyDetailsPage.jsx

```diff
@@ -78,7 +78,7 @@ const AgencyDetailsPage = () => {
     }, [agencyCode]);
 
     return (
-        <div className="usa-da__about-the-data__agencies-page">
+        <div className="usa-da__about-the-data__agency-details-page">
```

Comment:
```suggestion
        <div className="usa-da-about-the-data usa-da-about-the-data_agency-details-page">
```
I think this would be more true to BEM, with `usa-da-about-the-data` being a block for the page, and `agency-details-page` being a modifier that warrants slightly different styling. To me, `__` indicates an element of the block like a sidebar or header. (We could possibly go over this at the FE dev meeting)